lib/byron/src/Cardano/Wallet/Byron.hs

```diff
@@ -200,22 +202,23 @@ serveWallet
     serveApp socket = do
         withNetworkLayer nullTracer bp addrInfo versionData $ \nl -> do
             withNtpClient ntpClientTracer ntpSettings $ \ntpClient -> do
-                byronApi   <- apiLayer (newTransactionLayer @n) nl
-                icarusApi  <- apiLayer (newTransactionLayer @n) nl
-                startServer socket byronApi icarusApi ntpClient $> ExitSuccess
+                let pm = fromNetworkMagic $ networkMagic $ fst versionData
```

Comment:
maybe it would be worth to log some info here, `versionData` etc

Reply:
Done.